pkg/reaper/podreaper/podreaper.go

```diff
@@ -217,3 +225,81 @@ func (ctx *ReaperContext) validateArguments(args *Args) error {
 
 	return nil
 }
+
+func loadFakeAPI(ctx *ReaperContext) {
+	fakeNamespaces := []struct {
+		namespaceName string
+	}{
+		{
+			namespaceName: "namespace-1",
+		},
+		{
+			namespaceName: "namespace-2",
+		},
+		{
+			namespaceName: "namespace-3",
+		},
+	}
+
+	// Create fake namespaces
+	for _, c := range fakeNamespaces {
+		namespace := &v1.Namespace{ObjectMeta: metav1.ObjectMeta{
+			Name: c.namespaceName,
+		}}
+		ctx.KubernetesClient.CoreV1().Namespaces().Create(namespace)
+	}
+}
+
+func createFakePods(pods []FakePod, ctx *ReaperContext) {
```

Comment:
Why do we need this moved to `podreaper.go` it's the same package, we can keep this code under `podreaper_test.go`